src/arch/riscv/isa/templates/vector_mem.isa

```diff
@@ -175,6 +177,15 @@ Fault
     %(op_rd)s;
     %(ea_code)s;
 
+    MISA misa = xc->readMiscReg(MISCREG_ISA);
+    STATUS status = xc->readMiscReg(MISCREG_STATUS);
+    if (!misa.rvv || status.vs == VPUStatus::OFF) {
+        return std::make_shared<IllegalInstFault>(
```

Comment:
I think it makes sense to check the `VPUStatus` in execute, but I think the `misa.rvv` and `vill` should be checked at decode time.

Please correct me if I'm misunderstanding.

Reply:
Same as above.